db/migrate/20160414185708_create_container_label_tag_mappings.rb

```diff
@@ -0,0 +1,13 @@
+class CreateContainerLabelTagMappings < ActiveRecord::Migration[5.0]
+  def change
+    create_table :container_label_tag_mappings do |t|
+      t.string :labeled_resource_type
```

Comment:
Just to clarify, `labeled_resource_type` is not trying to a polymorphic association, 
it's just a string (or NULL) constraining the value of `resource_type` on the label.
[Motivation ex: one might want to map `zone` labels to tags only on container nodes and not all entities.  Though I expect it will be mostly unconstrained.]